Simonyi/Components/MTCS/MTDome.rst

```diff
@@ -1,6 +1,115 @@
+
+Although not part of MTDome, the Environmental Sensors Suite (ESS) keeps track of the environmental MTDome conditions. The MTDome also incorporates a high	 capacity HVAC system operated during the daytime to set the internal dome	 temperature to -1C below the expected start of night temperature.
+
+MTDome CSCs
+===========
+
+The MTDome has a dedicated Commandable SAL Component (CSC). The capacitor banks telemetry is collected via MTDome CSC. MTDomeTrajectory is the CSC in charge of enabling the MTDome following.
+
+
+MTDome Controller
+=================
+
+The Dome Control System (DCS) enable wireless communications. It is the interface to the facility lower enclosure which includes the azimuth bearing track, azimuth drive system, encoder system, slip ring and braking system. The Dome interfaces with the concrete lower enclosure through the azimuth support track, drive units, and slip rings. The MTDome CSC and GIS (Global Interlock System) communicate with the DCS (which also contains the DIS, Dome Interlock System) via ethernet. The fixed part of the DCS communicates with the rotating part of the DCS via WiFi. Currently, the MTDome Controller (DCS) is still under development.
+
+The DCS includes the following software subsystems:
+
+* AMCS - Azimuth Motion Control System which rotates the dome.
+* LWSCS - Light and Wind Screen Control System which will move up and down the Light and Wind Screen once it has been installed and placed under software control.
+* ApSCS - Aperture Shutter Control System which opens and closes the Aperture Shutter.
+* LCS - Louvers Control System which opens and closes the louvers.
+* ThCS - Thermal Control System which is in charge of keeping the temperature of the azimuth drives and other components within requirements.
+
+There are other software systems that the DCS reports the status for. They are called Control System to stay in line with the naming convention of the previous five but cannot be controlled via software. Those are:
+
+* CBCS - Capacitor Banks Control System which monitors the status of the capacitor banks. There are Watcher rules in place that will raise alarms in case the CBCS reports alarming situations or in case the azimuth motion reports errors. More Watcher alarms will be added when more MTDome parts come under software control.
+* CSCS - Calibration Screen Control System which monitors the status of the calibration screen hardware.
+* MonCS - Monitoring Control System which monitors various parts of the dome hardware.
```

Comment:
"....various parts of the dome hardware." --> Perhaps explicitly mention which parts